resources/cdk/sqs_lambda_to_batch_fargate/consumer_stack/consumer_stack.py

```diff
@@ -0,0 +1,202 @@
+            inline_policies={
+                "MyCustomPolicy": iam.PolicyDocument(
+                    statements=[
+                        iam.PolicyStatement(
+                            effect=iam.Effect.ALLOW,
+                            actions=[
+                                "logs:CreateLogGroup",
+                                "logs:CreateLogStream",
+                                "logs:PutLogEvents",
+                                "logs:DescribeLogStreams"
+                            ],
+                            resources=["arn:aws:logs:*:*:*"]
+                        )
+                    ]
+                )
+            },
+            managed_policies=[
+                iam.ManagedPolicy.from_aws_managed_policy_name(
+                    "job-function/SystemAdministrator"
+                ),
+                iam.ManagedPolicy.from_aws_managed_policy_name(
+                    "service-role/AmazonECSTaskExecutionRolePolicy"
+                )
+            ]
+        )
+
+        # Batch resources commented out due to bug: https://github.com/aws/aws-cdk/issues/24783.
+        # Using Alpha as workaround.
+
+        fargate_environment = batch_alpha.ComputeEnvironment(self, f"FargateEnv-{language_name}",
+            compute_resources=batch_alpha.ComputeResources(
+                type=batch_alpha.ComputeResourceType.FARGATE,
+                vpc=ec2.Vpc.from_lookup(self, "Vpc", is_default=True)
+            )
+        )
+
+        job_definition = batch_alpha.JobDefinition(self, f"JobDefinition-{language_name}",
+            container=batch_alpha.JobDefinitionContainer(
+                image=container_image,
+                execution_role=batch_execution_role
+            ),
+            platform_capabilities=[ batch_alpha.PlatformCapabilities.FARGATE ]
+        )
+
+        job_queue = batch_alpha.JobQueue(self, f"JobQueue-{language_name}",
+            compute_environments=[batch_alpha.JobQueueComputeEnvironment(
+               compute_environment=fargate_environment,
+               order=1
+            )]
+        )
+
+        # Define the Lambda function
+        function = _lambda.Function(self, f'SubmitBatchJob-{language_name}',
+                                    runtime=_lambda.Runtime.PYTHON_3_8,
+                                    handler='lambda_handler.lambda_handler',
+                                    role=execution_role,
+                                    code=_lambda.Code.from_asset('lambda'),
+                                    environment={
+                                        'LANGUAGE_NAME': language_name,
+                                        'JOB_QUEUE': job_queue.job_queue_arn,
+                                        'JOB_DEFINITION': job_definition.job_definition_arn,
+                                        'JOB_NAME': f'job-{language_name}'
+                                    }
+                                    )
```

Comment:
I would put these two closing symbols on the same line, rather than consecutive ones when they're alone, but that's not such a big deal.